docs/sources/flow/concepts/configuration_language.md

```diff
@@ -73,10 +73,6 @@ The most common expression is to reference the exports of a component like
 formed by merging the component's name (e.g., `local.file`), label (e.g.,
 `password_file`), and export name (e.g., `content`), delimited by period.
 
-For components that don't use labels, like
-`prometheus.exporter.unix`, only combine the component name with
-export name: `prometheus.exporter.unix.targets`.
```

Comment:
There is no longer any component which doesn't use a label; all of them require specifying a label.